.github/workflows/run-ci.yml

```
@@ -0,0 +1,19 @@
+name: Run CI
+
+on: push
+
+jobs:
+  'run-ci':
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+    # change this to (see https://github.com/ruby/setup-ruby#versioning):
+    # uses: ruby/setup-ruby@v1
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
```

I'm curious why `v1` is not sufficient here.

---

I borrowed the code from here.
https://github.com/actions/starter-workflows/blob/27a54d9d4c8f0dd4949d40837df6f690442aae42/ci/ruby.yml#L30

This code specifies this commit https://github.com/ruby/setup-ruby/commit/473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e but from the comment above it'd be better to change `v1` instead of specific commit.

```suggestion
      uses: ruby/setup-ruby@v1
```